app/src/time/timecontrol.cpp

```diff
@@ -32,13 +30,13 @@ bool TimeControl::updateTime(const int64_t elapsed_millis) noexcept {
     if (limits_.moves > 0) {
         if (moves_left_ == 1) {
             moves_left_ = limits_.moves;
-            time_left_ += limits_.time;
+            time_left_ += limits_.time + limits_.increment;
```

Comment:
you sure increments gets added again before the move is played once the timecontrol repeats? i dont think so

Reply:
hmm right, the increment is already added before the movesleft check, ill remove it

Reply:
removed